addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -222,6 +239,20 @@
 </table>
 </td></tr>
 </table>
+</body></html>
+        </template>
+
+        <template id="preview_with_padding">
```

Comment:
``notification_preview`` ? Not sure the "with padding" is useful (if someday we change the padding we would have to change the name :D ).